### PR TITLE
Feature/synchrony driver path

### PR DESCRIPTION
**Pull request by mattohope**

### CODE REVIEW

**Review by jlawton-adaptavist — COMMENTED** [review]
we will want to be mindful of making sure we don't have multiple jdbc drivers or this might not be able to determine which jar to use.

**Review by viktorkeremedchiev — APPROVED** [review]

File: share/avst-app/lib/atlassian/modify.d/71setup_dc [hunk 1]
```diff
@@ -175,7 +175,7 @@ EOF
         
         *)
             echo "Attempted to add DC Configuration for PRODUCT: ${PRODUCT} but this is not possible"
-            exit 50
+            exit 56
```
Comment (acerezoi) [inline share/avst-app/lib/atlassian/modify.d/71setup_dc:178]: Should the exit code 56 be 55? I haven't seen a 56 error code in the Readme

File: share/avst-app/lib/atlassian/modify.d/71setup_dc [hunk 2]
```diff
@@ -229,6 +229,6 @@ EOF
             ;;
         *)
             echo "Attempted to remove DC Configuration for PRODUCT: ${PRODUCT} but this is not possible"
-            exit 50
+            exit 56
```
Comment (acerezoi) [inline share/avst-app/lib/atlassian/modify.d/71setup_dc:232]: Should the exit code 56 be 55? I haven't seen a 56 error code in the Readme

File: README.md [hunk 3]
```diff
@@ -105,6 +106,17 @@ The following exit/return codes are used (as of 2014-10-09):
 * 46: Can't start service
 * 50: Can't set CROWD_SSO for unsupported product
 * 51: Missing mandatory variables for CROWD
+* 52: Can't set OKTA_SSO for unsupported product
+* 53: Missing mandatory variables for OKTA
+* 55: Missing mandatory variables for DC
+* 52: Can't setup DC for unsupported product
```
Comment (acerezoi) [inline README.md:112]: I put 2 comments below about a 56 code that maybe should be 55, but actually this duplicated 52 should be the 56 instead?

File: share/avst-app/lib/atlassian/modify.d/71setup_dc (hunk 1 above)
Comment (mattohope) [reply share/avst-app/lib/atlassian/modify.d/71setup_dc:178]: i had the readme value wrong (duplicated code 52), corrected now

File: share/avst-app/lib/atlassian/modify.d/71setup_dc (hunk 2 above)
Comment (mattohope) [reply share/avst-app/lib/atlassian/modify.d/71setup_dc:232]: i had the readme value wrong (duplicated code 52), corrected now

File: README.md (hunk 3 above)
Comment (mattohope) [reply README.md]: yes this was a copy'n'paste issue, i've corrected it

**Review by jlawton-adaptavist — APPROVED** [review]
looks good.

**Review by acerezoi — APPROVED** [review]

**Review by nmohamad-avst — APPROVED** [review]

**Pull request merged**